docs/src/surfaces/card/card.jsx

```diff
@@ -343,12 +348,13 @@ export default class ViewsTitleBar extends React.Component {
                     </Highlighter>
 
                     {/* Compact */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub>
+                    <Heading variant="h2">
                         Compact
-                        <Header.Subheader>
-                            A card can have it's padding reduced.
-                        </Header.Subheader>
-                    </Header>
+                    </Heading>
+
+                    <Typography variant="body1">
+                        A card can have it's padding reduced.
```

Comment:
"it's" => "its".  And capitalize Card.

Reply:
Fixed.

Reply:
"its" is fixed.  "card" is still not capitalized, but meh.  Resolving.